src/map/status.cpp

```diff
@@ -6323,8 +6323,10 @@ static unsigned short status_calc_str(struct block_list *bl, struct status_chang
 		str += sc->data[SC_ALMIGHTY]->val1;
 	if (sc->data[SC_ULTIMATECOOK])
 		str += sc->data[SC_ULTIMATECOOK]->val1;
+	if (sc->data[SC_ALL_STAT_DOWN])
+		str -= sc->data[SC_ALL_STAT_DOWN]->val2;
 
-	return (unsigned short)cap_value(str,0,USHRT_MAX);
+	return (unsigned short)cap_value(str,SHRT_MIN,USHRT_MAX);
```

Comment:
These stats can go below 0 now?

Reply:
Thanks for review. Atemo said yes it is.
This is KRo screenshot from Atemo.
![195630729-54a72ea9-ba57-482a-8845-23ab49d92e9f](https://user-images.githubusercontent.com/47050704/199064134-6cfd11b8-7a60-441d-aed7-ddc9f253caa0.jpg)

Reply:
Thanks, yeah I had talked to Atemo too who confirmed it as well. So the issue here is that the `status_get_<stat>` functions are `unsigned short` which means they won't return negative values, even with your cap adjustment. This will require a bit more of refactoring to get these changes in line as we will need to confirm all the other functions and variables are adjusted properly as well.

Reply:
Ohhh...! You are right. Then ```singed short``` would be okay...? 
Actually I can't expect what detail problems or variables will be arise.
Do you have any thoughts?

Reply:
Don't worry. I wouldn't expect to implement that behavior in this PR as you would lose the scope of this one. I would just revert these pieces that you changed here for the `cap_value()` and we can then merge this and in a separate PR we can clean up the functions to support negative values!

Reply:
Alright! Thanks.